TEAM_STRUCTURES.md

```diff
@@ -12,9 +12,11 @@
     - [IPFS GUI](#ipfs-gui)
     - [IPFS Infrastructure](#ipfs-infrastructure)
     - [Dev Team Enablement (QA, Testing, Support)](#qa-testing-and-dev-team-enablement)
-  - **Community, Education and Usage:**
-    - [Community, Events & Evangelism](#community-events-evangelism)
-    - [Documentation, Education and Websites](#documentation-education-and-websites)
+  - **[Community](#community):**
+    - Evangelism (Events, Social Media)
+    - Editorial (Blogs, Documentation)
+    - Education (ProtoSchool, Demos)
+    - Automation (Probots, etc)
```

Comment:
If these are not working Groups and just areas of focus of the Community Working Group, let's delete these bullets from the table of contents to not mislead into thinking that there are 4 WGs